### ISSUE TITLE
Is the variable OWLREADY2_FORMATS incomplete?

### ISSUE THREAD

**Issue by CasperWA** [emmo-repo/EMMOntoPy#219]
I am unsure whether this is intended, but it seems to me the `OWLREADY2_FORMATS` variable in `ontopy.utils` is incomplete, i.e., it doesn't reflect the values of the format mapping variable `ontopy.utils.FMAP`.

**Comment by francescalb** [comment]
No, one reflects the formats available in rdflib and the other what is avilable in owlready2. Emmontopy handles both but converting non-owlready2-accepted formats to an OWLREADY2_FOMAT with the help of rdflib.
